extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -250,6 +256,13 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     super.onCreativeRender(isVerifiedAmpCreative);
     injectActiveViewAmpAnalyticsElement(
         this, this.ampAnalyticsConfig, this.responseHeaders_);
+    const iframe = this.element.querySelector('iframe');
```

Comment:
Keep in mind we will want the same fix to apply to doubleclick given it was also not centering (although it does not suffer from the invalid format issue)